docs/userGuide/plugins/algolia.mbdf

```diff
@@ -0,0 +1,44 @@
+#### `algolia`: Enabling Algolia DocSearch
+
+This plugin allows you to use [Algolia DocSearch](https://community.algolia.com/docsearch/) for your site.
+
+To enable it, add `algolia` to your site's plugins, and supply the required options via the `pluginsContext`.  
+
+Name | Type | Default | Description
+---- | ---- | ------- | ------
+apiKey | `String` || The API key for your site's Algolia DocSearch setup
+indexName | `String` || The index name for your site's Algolia DocSearch setup
+algoliaOptions | `Object` | `{}` | A JSON object specifying [additional options for DocSearch](https://community.algolia.com/docsearch/behavior.html#algoliaoptions)
+debug | `Boolean` | `false` | Whether to turn on debug mode to allow inspection of CSS styles for the dropdown
+
+```json
+site.json
+{
+  ...
+  "plugins": [
+    "algolia"
+  ],
+  "pluginsContext": {
+    "algolia": {
+      "apiKey": INSERT_ALGOLIA_API_KEY,
+      "indexName": INSERT_ALGOLIA_INDEX_NAME,
+      "algoliaOptions": {
+        "hitsPerPage": 10,
+      },
+      "debug": false
+    }
+  }
+}
+```
```

Comment:
It's good to give an example using pseudo-real values so that the reader get more confidence by comparing her version with the example.
Also, all these values compulsory? If not, indicate which ones are optional.

Reply:
I've updated it to use the api key and index name for `se-book`.

Reply:
Understand that our keys are visible to the public on the original site anyway, still feels a bit weird to use real values though. 😅 Consider using a make up one?

Reply:
Yes, modify it slightly so that it looks like the real one but is not.

Reply:
We could also use the values from [DocSearch's documentation](https://community.algolia.com/docsearch/dropdown.html#testing), since they are suggested for testing purposes.

Reply:
Updated to use the example values on DocSearch!